test/mock/mock-management-plane-api.yaml

```diff
@@ -0,0 +1,170 @@
+openapi: 3.0.2
+info:
+  title: Management Plane API
+  version: 0.0.1
+security:
+  - Bearer: []
+paths:
+  /instances/{instanceId}/files/:
+    get:
+      summary: Return all files metadata for a data plane instance.
+      operationId: GetInstanceFilesMetadata
+      parameters:
+        - in: header
+          name: tenantId
+          schema:
+            type: string
+            format: uuid
+          required: false
+        - in: path
+          name: instanceId
+          schema:
+            type: string
+          required: true
+          description: Instance ID
+          example: "aecea348-62c1-4e3d-b848-6d6cdeb1cb9c"
+      responses:
+        '200':
+          description: Files metadata for a data plane instance.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/FilesResponse'
+  /instances/{instanceId}/files/{filePath}:
+    get:
+      summary: Download a specific file for a data plane instance.
+      operationId: GetInstanceFile
+      parameters:
+        - in: header
+          name: tenantId
+          schema:
+            type: string
+            format: uuid
+          required: false
+        - in: path
+          name: instanceId
+          schema:
+            type: string
+          required: true
+          description: Instance ID
+          example: "aecea348-62c1-4e3d-b848-6d6cdeb1cb9c"
+        - in: path
+          name: filePath
+          schema:
+            type: string
+          required: true
+          description: file location
+          example: "/etc/nginx/nginx.conf"
```

Comment:
I believe I brought this up before but what happens if we pass an absolute path to `filePath`? Since we are passing it in the URL, does the leading slash get omitted?

Reply:
no the leading slash is still ther. Here is an example with the file path encoded in the url:
`api/v1/instances/32/files/%2Fetc%2Fnginx%2Fnginx.conf?encoded=true`

Reply:
Should we add note in the description to URL encode the `filePath`?

Reply:
sure let me add it there